include/gridtools/stencil_composition/frontend/cartesian/accessor.hpp

```diff
@@ -105,18 +117,23 @@ namespace gridtools {
             static constexpr intent intent_v = Intent;
             using extent_t = Extent;
 
+            template <class E = Extent,
+                std::enable_if_t<E::iminus::value <= 0 && E::iplus::value >= 0 && E::jminus::value <= 0 &&
```

Comment:
As this is user api, maybe a static_assert with message would be better to indicate what the user's mistake is.

Reply:
Good idea! Actually this ctor is redundant. I have add more accurate static_assert